index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Technical Documentation Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>Product Landing Page</header>
+        <div>
+          <a href="#introduction" class="nav-link">Introduction</a>
+          <a href="#What_you_should_already_know" class="nav-link"
+            >What you should already know</a
+          >
+          <a href="#JavaScript_and_Java" class="nav-link"
+            >JavaScript and Java</a
+          >
+          <a href="#Variables" class="nav-link">Variables</a>
+          <a href="#Function_declarations" class="nav-link"
+            >Function declarations</a
+          >
+        </div>
+      </nav>
+
+      <section class="main-section" id="introduction">
+        <header>introduction</header>
+        <p>
+          JavaScript is a cross-platform, object-oriented scripting language. It
+          is a small and lightweight language. Inside a host environment (for
+          example, a web browser), JavaScript can be connected to the objects of
+          its environment to provide programmatic control over them.
+        </p>
+        <p>
+          JavaScript contains a standard library of objects, such as Array,
+          Date, and Math, and a core set of language elements such as operators,
+          control structures, and statements. Core JavaScript can be extended
+          for a variety of purposes by supplementing it with additional objects;
+          for example:
+        </p>
+        <ul>
+          <li>
+            Client-side JavaScript extends the core language by supplying
+            objects to control a browser and its Document Object Model (DOM).
+            For example, client-side extensions allow an application to place
+            elements on an HTML form and respond to user events such as mouse
+            clicks, form input, and page navigation.
+          </li>
+          <li>
+            Server-side JavaScript extends the core language by supplying
+            objects relevant to running JavaScript on a server. For example,
+            server-side extensions allow an application to communicate with a
+            database, provide continuity of information from one invocation to
+            another of the application, or perform file manipulations on a
+            server.
+          </li>
+        </ul>
+      </section>
+      <section class="main-section" id="What_you_should_already_know">
```

Comment:
-  your <header> tag should wrap the toolbar or the nav not the sections title. use heading(h2,h3,h4,...)tags instead.
see the last comment for reason.

Reply:
- your <header> tag should wrap the toolbar or the nav not the title of sections. you could use  the heading(h2,h3,h4,...) tags instead.
see the last comment for reason.

Reply:
- i think your header tag should wrap the toolbar or the nav not the title of sections. you could use the heading(h2,h3,h4,...) tags instead.

see the last comment for reason.